src/macros.jl

```diff
@@ -448,12 +445,12 @@ macro constraint(args...)
                 try
```

Comment:
This whole `coeftype` thing doesn't seem well thought-out. Maybe we should get rid of it.